composer.json

```diff
@@ -89,6 +89,7 @@
         "willdurand/hateoas-bundle": "^2.0"
     },
     "conflict": {
+        "doctrine/doctrine-bundle": "<2.0 || ^3.0",
```

Comment:
cause it has been moved on dev requirement, we have to ensure the right version is installed.